backend/src/main/java/com/example/backend/use_case/display_stops/StopsService.java

```diff
@@ -0,0 +1,19 @@
+package com.example.backend.use_case.display_stops;
+
+import com.example.backend.data_access.stop.StopDataAccessInterface;
+import com.example.backend.entity.Location;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+
+@Service
+public class StopsService {
+    @Autowired
+    private StopDataAccessInterface stopDataAccessInterface;
```

Comment:
name needs to be changed to stopDataAccessObject

Reply:
Done.